cli/lib/kontena/cli/config.rb

```diff
@@ -79,9 +79,41 @@ def extract_token!(hash={})
         )
       end
 
-      # Load configuration from default location ($HOME/.kontena_client.json)
+      def move_config
+        if File.exist?(old_default_config_filename)
+          ensure_config_path
+          require 'fileutils'
+          FileUtils.mv(old_default_config_filename, default_config_filename)
+          at_exit do
+            # Need to do at exit, otherwise it will pop up in the first place the config is accessed,
```

Comment:
The `at_exit` seems a bit extra... is this still still needed with the class-level `require_current_master`?